lib/platforms.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
             '<=0.9.2': ['nw.exe', 'ffmpegsumo.dll', 'icudt.dll', 'libEGL.dll', 'libGLESv2.dll', 'nw.pak'],
             '>0.9.2': ['nw.exe', 'ffmpegsumo.dll', 'icudtl.dat', 'libEGL.dll', 'libGLESv2.dll', 'nw.pak', 'locales']
         },
-        versionNameTemplate: 'v${ version }/node-webkit-v${ version }-win-ia32.zip'
+        versionNameTemplate: 'v${ version }/nwjs-v${ version }-win-ia32.zip'
```

Comment:
Wont this break because the pre 0.12.0 versions of node webkit are still called `node-webkit` not `nwjs`? Look in http://dl.nwjs.io/v0.11.5/ for example.